src/main/resources/db/migration/V20180209195900__move-ds-schemas.sql

```diff
@@ -0,0 +1,52 @@
+ALTER TABLE data_sources_data
+ADD COLUMN published BOOLEAN DEFAULT FALSE NOT NULL;
+
+UPDATE data_sources_data set published = TRUE;
+
+ALTER TABLE data_sources_data ALTER organization DROP NOT NULL;
+ALTER TABLE data_sources_data ALTER model_type DROP NOT NULL;
+
+ALTER TABLE datanodes ALTER name DROP NOT NULL;
+ALTER TABLE datanodes ALTER description DROP NOT NULL;
```

Comment:
let's add trigger (?) here

Reply:
please clarify trigger on what?
after adding 'published' column for data-node table - the situation will be the same as with ds checker.